applications/operators/olm-creating-policy.adoc

```diff
@@ -0,0 +1,39 @@
+[id="olm-creating-policy"]
+= Creating policy for Operator installations and upgrades
+include::modules/common-attributes.adoc[]
+:context: olm-creating-policy
+
+toc::[]
+
+Operators can require wide privileges to run, and the required privileges can
+change between versions. Operator Lifecycle Manager (OLM) runs with
+`cluster-admin` privileges. By default, Operator authors can specify any set of
+permissions in the ClusterServiceVersion (CSV) and OLM will consequently grant
+it to the Operator.
+
+Cluster administrators should take measures to ensure that an Operator cannot
+achieve cluster-scoped privileges and that users cannot escalate privileges
+using OLM. One method for locking this down requires cluster administrators
+auditing Operators before they are added to the cluster. Starting in
+{product-title} 4.2, however, administrators are provided tools for determining
+and constraining which actions are allowed during an Operator installation or
+upgrade.
+
+By associating an OperatorGroup with a service account with a set of privileges
+granted to it, cluster administrators can set policy on Operators to ensure they
```

Comment:
This sentence isn't clear. Not sure what you are associating with what.

Reply:
Updated to: "By associating an OperatorGroup with a service account that has a set of
privileges granted to it, [...]"